### DISCUSSION TITLE
💢【大家理智，不要被带偏了】M国也有次贷危机，信贷危机....

### DISCUSSION THREAD

**Discussion by hyc8801** [WeNeedHome/SummaryOfLoanSuspension#614] [Q&A]
烂尾、停贷，本身大多数原因是经济问题，当然也不排除有贪腐问题。

但回到问题本身，始终是经济问题，所以各位不要被不明势力带入坑里面去了。

大家要的只是解决烂尾的问题。请理智讨论~~~~~

将话题带偏对于对于解决烂尾并无好处，将矛盾转移的人大家要擦亮眼睛~~~~💥💥💥💥

**Comment by JACWATA** [comment]
先让墙内能发这些内容了，再说和政治无关这种话

**Reply by hyc8801** [reply]
墙内不能发的内容多了，涩情都能被你扯上政治，从大局出发是稳定人心，如果按M国的情况，当人心惶惶的时候，就会出现真正的次贷危机

**Reply by randufeng** [reply]
请先了解下本国停贷者的无限责任制,再说M国

**Reply by hyc8801** [reply]
所以说这是经济问题，资本家的问题~ 有些人转移矛盾和问题本质就是居心叵测

**Reply by ckckckckk** [reply]
十个人一百个人是经济问题，一百万一千万就是政治问题！

**Reply by purpose1113** [reply]
中国停贷者的无限责任制,  美国是有限责任制， 果然是中国特色  ，人为刀俎我为鱼肉

**Reply by JACWATA** [reply]
> 请先了解下本国停贷者的无限责任制,再说M国

还有最高的房贷、最贵的首付，动不动就想掩盖

**Reply by randufeng** [reply]
我觉得不是转移,是陈述事实,先了解事实再定性责任

**Reply by hyc8801** [reply]
那你这么多，M过爆发次贷危机，你怎么不反M？

**Reply by JACWATA** [reply]
> 那你这么多，M过爆发次贷危机，你怎么不反M？

别人不准说不准聊了吗？

**Reply by TALK1949** [reply]
> 那你这么多，M过爆发次贷危机，你怎么不反M？

美国次贷危机关业主啥事？因为别人家也死人了，所以被车撞死司机就不用负责？

**Reply by ckckckckk** [reply]
> 那你这么多，M过爆发次贷危机，你怎么不反M？

兄嘚，你这才是在转移矛盾，虽然我没买房，没贷款，但如果你看一下环华十年拍的西安易合坊的业主们，你如果还能觉得这是经济的问题？那我觉得没有争辩的意义了

**Reply by yangtuqiang** [reply]
都反墙了，还和你玩老爷那一套么

**Reply by zhuuvzhi** [reply]
> 那你这么多，M过爆发次贷危机，你怎么不反M？

M国是啥？😂

**Reply by zongjiangU** [reply]
> 所以说这是经济问题，资本家的问题~ 有些人转移矛盾和问题本质就是居心叵测
什么是政治，什么是经济，告诉我好吗，敲泥麻。

**Reply by Miniluv8964** [reply]
美国就美国，色情就色情，到墙外了还不忘给自己阉割一下，果然太监就是太监，生理上的鸡巴容易有，心理上的鸡巴难有

**Comment by 23lc** [comment]
停贷和政治有关系，贪腐问题就是政治问题。
你要业主们齐心协力做成一件事情，这本身就是非常严肃的政治话题。
但是，某些虫豸讨论的所谓“政治问题”，本质上就是娱乐八卦屎尿屁。我们应当跟这种人坚决划清界限。

**Reply by hyc8801** [reply]
警惕 某些人 浑水摸鱼，转移矛盾即可。

**Reply by clleo888** [reply]
转移矛盾有谁比共产党更熟手? 美国又有人死人了....

**Reply by lance111l** [reply]
停贷是经济问题  当然不是政治问题   开发商资金运作不行  倒闭了   美国倒闭的公司少吗

**Comment by JACWATA** [comment]
五毛、粉蛆一枚鉴定完毕

**Reply by hyc8801** [reply]
只是提醒大家不要被带偏，回到问题本身，从未对他人进行攻击

**Reply by ppwq2022** [reply]
> 五毛、粉蛆一枚鉴定完毕

那你是行走的50万 4V，还是又蠢又坏

**Reply by ppinkiris** [reply]
昂撒的狗？要当自己当，别在国内恶心人

**Reply by wangsir003** [reply]
你个智障玩意

**Comment by hyc8801** [comment]
警惕某些人 偷换概念，转移矛盾即可！

**Reply by mutoto321** [reply]
虽然和你的观点不同，但还是提醒你一下暴露手机号了改了吧，你相当于在裸奔

**Reply by hyc8801** [reply]
已更改了~只是提醒大家理智点罢了，自身也是支持言论自由的，试问有谁不爱自由

**Reply by mand-byte** [reply]
政治就是如何分蛋糕的,这蛋糕分的有问题.

**Comment by C-Joey** [comment]
M国是什么东西 是理想国吗
![85fb4b039ee332e5](https://user-images.githubusercontent.com/48274987/179190717-47bfc33c-f391-4503-89da-20b31e8c8efe.jpg)

**Comment by night-yang** [comment]
地方政府需要钱=》需要开发商拿地=》高周转开发商需要钱拿地=》监管名存实亡，用来买房的看，不是给银行用的（住建局批领条子宽的一批）^房地产下行=》房子不好卖，收入无法覆盖利息=》烂尾跑路

**Reply by night-yang** [reply]
再看看谁从里面拿钱了，地方政府，银行，开发商，看看谁出钱了，业主，再看看各自的责任，政府超然，不直接参与交易，同时下达了监管条例（名存实亡），银行虽然有监管责任，但是肯定能找出一堆合法文件，同时由于贷款合同和购房合同是分开的，除了说一句银行失责，还真没别的办法，大不了相关人员坐牢，追缴贷款只认贷款合同，开发商责任最大，但是为什么权力最小理应受到各方监管的开发商，能犯这么多的错？这和相关方面纵容肯定有关系，业主的责任？大概是不该买房吧，不该相信期房资金监管条例可以真正执行吧

**Reply by night-yang** [reply]
不是说爱国就要包容这个国家的一切，都知道毛主席说过什么，我就不重复了，你爱他就要大胆的指出他做错的地方，如果为人民服务不是说说而已，自然会考虑人民的利益
不从本质解决问题，只解决嘴上问题，只会越来越麻烦
法律如果不能保证各方利益，而是持续的让一方失血，那老百姓只能靠着按闹分配维护自己的权益了

**Reply by night-yang** [reply]
现在还是经济问题，但是如果不解决，还追贷，很快就会变成政治问题。
而且就算你解决了，但是后续法律法规跟不上，也只会告诉老百姓按闹分配是王道。
不从土地财政这个制度下手，这个问题迟早变成政治问题

**Reply by radiateboy** [reply]
睿智，精辟

**Reply by Ningmeng-yp** [reply]
这才是实话

**Reply by itsoursduty** [reply]
你跟他们讲道理 他们只会扣帽子 说你50w 岛民 。明明反对，抗议，才是改善现状的唯一方式，结果小粉红们依然翘首以盼期待老爷们从兜里掏出钱来还给屁民 。

**Reply by ruo-xi** [reply]
我爱国 爱的是这个国家的无产阶级，而不是资本家 特权阶级。

**Comment by taopeng-life** [comment]
要警惕这个人，坏得很，不要被它带偏了
国内这些事都不能说，说了就删除，幸福就对了

**Reply by JACWATA** [reply]
> 要警惕这个人，坏得很，不要被它带偏了 国内这些事都不能说，说了就删除，幸福就对了

别问，问就是感恩

**Reply by HorLzon** [reply]
幸好没有米买房

**Comment by TALK1949** [comment]
在街边随便溜达都能看到各种政治标语，却总有人要强调在中国可以不谈政治，自欺欺人。

**Reply by clleo888** [reply]
中共就是会骗, 全人类都知道的!

**Comment by luowenfeng** [comment]
没毛病啊, 美国次贷危机的时候大家可以骂小布什, 中国烂尾楼停贷为啥不让骂包子?

**Reply by zhaomingtao-fighting** [reply]
包子是谁？

**Reply by miaodi** [reply]
> 包子是谁？

剧毒术士徐俊平你不会不认识吧？

**Reply by j38472** [reply]
美资本大佬大都是犹太人，你咋不在美国骂犹太人呢？尼哥打砸抢烧，你在不敢指着尼哥喊他尼哥呢？？？？？？？？

**Comment by Accelerator6666** [comment]
倒车请注意，倒车请注意……

**Comment by radiateboy** [comment]
谁能把业主问题解决了（任何手段都行），政治不政治无所谓你随便说。我给你俩舔。
5毛党要带节奏就带呗，墙外东西多了，翻不起什么大浪，要干大事你不在中南海某个职务，怕你连小区门都出不去。

**Reply by hyc8801** [reply]
是的，主要是解决问题

**Reply by redtime1213** [reply]
> 是的，主要是解决问题
解决尼玛 不停贷 谁TM会替你解决问题 傻逼东西 还抱有幻想

**Comment by zcaaron** [comment]
你只要有问题牵涉到政府，就是牵涉到政治，是绕不开的。

**Comment by nitamademalegebi** [comment]
你生活的方方面面都和政治有关系，你个二傻子还在这里bb

**Reply by davidliu1645** [reply]
怕是中共的水军

**Comment by tanuk1997** [comment]
经费维稳🐕一名，鉴定完毕

**Comment by jiushi555** [comment]
第一：M国并不是全款预售制，银行是非常严格的监管购房资金，因为银行都是私立的，他们对风控把控的很严格。购房者交首付，银行只会把5%的钱给到开发商、中国则是全款。
第二：M国遇到开发商烂尾，由于自己已贷出部分资金，客户选择停止还贷，银行收回抵押物进行拍卖，拍卖的钱，只需要购房者还贷的那部分，zhong是全款。
第三：M国的房价，不需要掏空六个钱包，谢谢

**Reply by nldxtd** [reply]
都出来了还打缩写呢

**Reply by baoang** [reply]
> 都出来了还打缩写呢

這是國內發貼的標配。我懷疑如果真打成美國，牆內翻出來的一部人可能還會想一下，美國在哪個洲？可能還不如M國或美麗國這樣的“代詞”來得直接易懂。

**Reply by lance111l** [reply]
M国的次贷危机  是  "风控把控的很严格"?

**Comment by blacklion-spec** [comment]
吃饭拉屎，都是政治，生活就离不开政治

**Comment by Farconquer** [comment]
你猜猜美国次贷危机是谁救的？😁救美国就是救中国是谁说的

**Comment by Universe-Wong** [comment]
sb5m，什么都是别人带的都是别人的错是吧

**Comment by Edan233** [comment]
虽然但是 经济离不开政治

**Comment by BigRoseAndBoss** [comment]
中国没有经济问题，只有政治问题

**Comment by stephenzhao** [comment]
“这里不做任何非数据性质等其他一系列讨论，只是统计一个数据，有错就纠改，无其他任何含义，别的请勿多言！禁止政治敏感话题！” 
谨记项目初衷

**Comment by King8611** [comment]
看来红码也是财政问题

**Comment by xiyuehua** [comment]
确实是经济问题，是什么造成了经济问题？疫情全世界都解封发展经济，咱们封着，但是咱们GDP增速还是牛啊，失业变灵活就业，封锁变动态清零，各地财政透支还要做核酸检测，医保亏空，银行都暴雷了，那你告诉我是啥造成了经济问题？

**Comment by 5EAR** [comment]
公公周末没有双休吗

**Comment by NiceAir** [comment]
大概看了下，带节奏的很多都是刚创建出来的小号。明明这个项目是用来统计烂尾楼停贷相关事情的，可那些人故意乱带节奏，骂政府、骂共产党、骂习近平、各种带节奏，真恶心

**Reply by k624035508** [reply]
卖地的最大受益人就是政府，自己赚了大头出现烂尾楼还不能骂了？

**Reply by volans19** [reply]
局外人帮奴隶骂奴隶主，奴隶说，别骂了别骂了，你不让他们强奸我你真恶心

**Reply by itsoursduty** [reply]
新号多 说明大家的自我保护意识还是挺好的 谁傻逼到用原来的账号裸奔来这骂中共？ 你这种人永远意识不到矛盾的根源，你只要你还相信中共一天，铁拳就永远在你的头上摇摇欲坠 ，到时候希望你还能像今天一样如此虔诚

**Comment by xiaer93** [comment]
（1）必须明确政治生活的作用。政治生活与经济生活、文化生活存在着相辅相成的关系。政治生活的内容更多地涉及社会公共管理和公共利益的问题。（2）政治生活在人民生活中不可或缺。


百度百科搜到的，政治生活。

**Comment by daxiazilong** [comment]
千万不要说他们美爹 比挖祖坟仇恨还大呢

**Reply by frshgrass** [reply]
其实我们根本无所谓美国怎么样 亡国灭种都关我屁事。但是我的半辈子积蓄和下半生钱  拿大头的政府 没有旅行监管责任 让我人房两空。我就吊你老母 别说我说脏话 他们也就配这些

**Comment by Speedor** [comment]
将矛盾转移的人大家要擦亮眼睛（除了最后一句比较比较认同，其他的么，呵呵）

**Comment by GitHeP** [comment]
推荐你看一本书 《血稠定律》

**Reply by K641645** [reply]
血酬定律

**Comment by mehhss** [comment]
美国也有次贷危机，所以赵立坚，华春莹和耿爽都在美国买房并把孩子都送到美国，还有习近平，他也送女儿去了美国

**Reply by volans19** [reply]
为什么不送到朝鲜伊朗俄罗斯呢，巴基斯坦也行

**Reply by shawn-yee** [reply]
因为虎毒不食子

**Comment by tangleiNUAA** [comment]
这个列表里面有不准确的内容，人都交房了，好些业主都搬进去住了，也在这个列表里面。建议仓库维护者还是甄别一下信息的准确性。

**Reply by baoang** [reply]
可能只有让这个信息不准确了，才会没有人信服这个repo，才会让这个repo真正的烂尾，从而引发不了哪怕一丝涟漪。

**Comment by Wesleylr** [comment]
众所周知资本主义的危机是完全无法解决的严重弊病，社会主义的危机是在艰难探索中成长
   ——选自中学教科书

**Comment by miaodi** [comment]
支蛆的必杀，whataboutism。

**Reply by Flywolfs** [reply]
狗日的傻逼

**Comment by volans19** [comment]
低等级五毛蛆，鉴定完毕，连带节奏都不会，说着不带偏，紧跟着又说美国，好好睁眼看看所有帖子，有谁夸过一次美国没，反向给党舔腚也不带这么着急的

**Comment by hrsgn** [comment]
这个有网友总结过，美国是次贷危机，我们是优贷危机，放一起比较，是在自打脸。

**Reply by toxin08x** [reply]
美国次贷危机是低信用购房者薅银行羊毛把它薅破产了，自己停贷破产7年后不良信用记录清零又是一条好汉。而我国的停贷潮则是韭菜被三方薅得根都要没了，以自焚的形式抗议，实在太惨烈了。

**Comment by haoww** [comment]
看完我都想为人民服务哈哈😂

**Comment by haoww** [comment]
韭菜割到起飞

**Comment by Ventery** [comment]
建议1450们下次带节奏换个有repo的号

**Comment by Siryedu** [comment]
学过经济学吗？巧了，我恰好是经济学专修的程序爱好者，谁告诉你经济和政治没关系的？什么叫只是经济问题？

**Comment by syyxsxx** [comment]
楼主傻鸟，M国次贷危机，断贷就不追责了，我们国家呢？我们是馒头被抢了，还要当奴隶，不当就把人埋了

**Comment by frshgrass** [comment]
搞笑呢 国内这高房价不就是政府收的进城税，结果收钱不办事 还把监管资金给弄没。自己躲在体制内吃香喝辣岁月静好。美国有占领华尔街 既然你要求对等 你是不是支持我们也可以。权益是斗争来的 要是东西被抢了 哼一声也不敢 只会被欺负得更惨淡。

大家用力的骂 高声得骂 要让他们知道 兔子急了也咬人 人民也是有力量的。

**Comment by mathLjtao** [comment]
鲁迅《聪明人、傻子和奴才》

奴才总不过是寻人诉苦。只要这样，也只能这样。有一日，他遇到一个聪明人。

“先生!”他悲哀地说，眼泪联成一线，就从眼角上直流下来。“你知道的。我所过的简直不是人的生活。吃的是一天未必有一餐，这一餐又不过是高粱皮，连猪狗都不要吃的，尚且只有一小碗……。”

“这实在令人同情”聪明人也惨然说。

“可不是么!”他高兴了。“可是做工是昼夜无休息的：清早担水晚烧饭，上午跑街夜磨面，晴洗衣裳雨张伞，冬烧汽炉夏打扇。半夜要煨银耳，侍候主人耍钱；头钱[2]从来没分，有时还挨皮鞭……。”

“唉唉……。”聪明人叹息着，眼圈有些发红，似乎要下泪。

“先生!我这样是敷衍不下去的。我总得另外想法子。可是什么法子呢？……。”

“我想，你总会好起来……。”

“是么？但愿如此。可是我对先生诉了冤苦，又得你的同情和慰安，已经舒坦得不少了。可见天理没有灭绝……。”

但是，不几日，他又不平起来了，仍然寻人去诉苦。

“先生!”他流着眼泪说“你知道的。我住的简直比猪窠还不如。主人并不将我当人；他对他的叭儿狗还要好到几万倍……。”

“混帐!”那人大叫起来，使他吃惊了。那人是一个傻子。

“先生，我住的只是一间破小屋，又湿，又阴，满是臭虫，睡下去就咬得真可以。秽气冲着鼻子，四面又没有一个窗……。”

“你不会要你的主人开一个窗的么？”

“这怎么行？……”

“那么，你带我去看看!”

傻子跟奴才到他屋外，动手就砸那泥墙。

“先生!你干什么？”他大惊地说。

“我给你打开一个窗洞来。”

“这不行!主人要骂的!”

“管他呢!”他仍然砸。

“人来呀!强盗在毁咱们的屋子了!快来呀!迟一点可要打出窟窿来了!……”他哭嚷着，在地上团团地打滚。

一群奴才都出来了，将傻子赶走。

听到了喊声，慢慢地最后出来的是主人。

“有强盗要来毁咱们的屋子，我首先叫喊起来，大家一同把他赶走了。”他恭敬而得胜地说。

“你不错。”主人这样夸奖他。

这一天就来了许多慰问的人，聪明人也在内。

“先生。这回因为我有功，主人夸奖了我了。你先前说我总会好起来，实在是有先见之明……。”他大有希望似地高兴地说。

“可不是么……。”聪明人也代为高兴似的回答他。

一九二五年十二月二十六日

**Comment by tacksonfu** [comment]
你可真的一个机灵鬼

**Comment by shawn-yee** [comment]
非洲人穷的没饭吃了
中东人一年四季打仗
有吃有穿，没有战乱，我们真是太幸福了，感恩，泪目。🤣

**Comment by ghost** [comment]
所以你这种人如果哪天挨了铁拳也真是福报了

**Comment by 505Games-oss** [comment]
不想活命的请举手......没有~没有~没有~没有~通过！！！！

**Comment by Cegazhos** [comment]
经济问题？买家给的钱因为经济危机不见了是吧？
明明监管问题扯什么经济问题

**Comment by Ventery** [comment]
题主，这里台🐸子和反共脑残比较多，不必跟他们一般见识。他们就不是来认真讨论的

**Reply by ghost** [reply]
你就是个鸡巴毛+傻逼舔狗一个😁真以为台湾人会关心你这点破b事儿吗😆

**Reply by Cegazhos** [reply]
> 你就是个鸡巴毛+傻逼舔狗一个😁真以为台湾人会关心你这点破b事儿吗😆

没有脑子就算了 眼睛也没有？建议回炉重造

**Reply by ghost** [reply]
> > 你就是个鸡巴毛+傻逼舔狗一个😁真以为台湾人会关心你这点破b事儿吗😆
> 
> 
> 
> 没有脑子就算了 眼睛也没有？建议回炉重造

哇哦，粉蛆急了，开始咬人了😆

**Reply by Cegazhos** [reply]
> > > 你就是个鸡巴毛+傻逼舔狗一个😁真以为台湾人会关心你这点破b事儿吗😆
> > 
> > 
> > 没有脑子就算了 眼睛也没有？建议回炉重造
> 
> 哇哦，粉蛆急了，开始咬人了😆

经典自爆，一骂台湾狗就反咬粉红。世上只有你妈的批不黑？

**Reply by frshgrass** [reply]
你是不是煞笔啊！洗脑洗得臭煞笔！自己利益都不关心！就听🐶党挑拨！你说台湾狗我还说台湾🐷呢！他妈的这和🐶党骑在我们头上作威作福有什么关系。吗的！什么一致对外！臭煞笔就自己人被欺负了！还想什么一致对外！该剥削你到死！洗脑得基本的逻辑都没有的！

**Reply by ghost** [reply]
> > > > 你就是个鸡巴毛+傻逼舔狗一个😁真以为台湾人会关心你这点破b事儿吗😆
> 
> > > 
> 
> > > 
> 
> > > 没有脑子就算了 眼睛也没有？建议回炉重造
> 
> > 
> 
> > 哇哦，粉蛆急了，开始咬人了😆
> 
> 
> 
> 经典自爆，一骂台湾狗就反咬粉红。世上只有你妈的批不黑？

嗯是呗。你妈的逼是黑的，粉B的时候都献给党了，然后就被操黑了呗。

**Reply by ghost** [reply]
> 你是不是煞笔啊！洗脑洗得臭煞笔！自己利益都不关心！就听🐶党挑拨！你说台湾狗我还说台湾🐷呢！他妈的这和🐶党骑在我们头上作威作福有什么关系。吗的！什么一致对外！臭煞笔就自己人被欺负了！还想什么一致对外！该剥削你到死！洗脑得基本的逻辑都没有的！

没办法👐，有些人天生当奴才的料，我等看官乐呵乐呵就好，看他们那副义愤填膺、如丧考妣的样子就开心🥳

**Reply by Cegazhos** [reply]
> > > > > 你就是个鸡巴毛+傻逼舔狗一个😁真以为台湾人会关心你这点破b事儿吗😆
> > 
> > 
> > > >
> > 
> > 
> > > >
> > 
> > 
> > > > 没有脑子就算了 眼睛也没有？建议回炉重造
> > 
> > 
> > > 
> > 
> > 
> > > 哇哦，粉蛆急了，开始咬人了😆
> > 
> > 
> > 经典自爆，一骂台湾狗就反咬粉红。世上只有你妈的批不黑？
> 
> 嗯是呗。你妈的逼是黑的，粉B的时候都献给党了，然后就被操黑了呗。

台湾傻狗😁好急 好急

**Reply by frshgrass** [reply]
你这个精神胜利法好可以啊！明明大家都是大陆人！我也无所谓你怎么诅咒台湾，甚至我可以为了好玩和你一起骂台湾！但是让你正视国内党的独裁奴隶人民！你就说台湾！你不是脑子洗坏了就是欺软怕硬的垃圾人。你不觉得自己很可悲吗？

**Reply by frshgrass** [reply]
> > > > > > 你就是个鸡巴毛+傻逼舔狗一个😁真以为台湾人会关心你这点破b事儿吗😆
> 
> > > 
> 
> > > 
> 
> > > > >
> 
> > > 
> 
> > > 
> 
> > > > >
> 
> > > 
> 
> > > 
> 
> > > > > 没有脑子就算了 眼睛也没有？建议回炉重造
> 
> > > 
> 
> > > 
> 
> > > > 
> 
> > > 
> 
> > > 
> 
> > > > 哇哦，粉蛆急了，开始咬人了😆
> 
> > > 
> 
> > > 
> 
> > > 经典自爆，一骂台湾狗就反咬粉红。世上只有你妈的批不黑？
> 
> > 
> 
> > 嗯是呗。你妈的逼是黑的，粉B的时候都献给党了，然后就被操黑了呗。
> 
> 
> 
> 台湾傻狗😁好急 好急
>
你这个精神胜利法好可以啊！明明大家都是大陆人！我也无所谓你怎么诅咒台湾，甚至我可以为了好玩和你一起骂台湾！但是让你正视国内党的独裁奴隶人民！你就扣帽子说是台湾人！大哥劳资大陆人！无所谓你怎么骂台湾！关我屁事！但是现在国内威权干的不是人事！颠倒黑白！你还在转移矛盾！你不是脑子洗坏了就是欺软怕硬的垃圾人。你不觉得自己很可悲吗？

**Reply by Cegazhos** [reply]
> 你这个精神胜利法好可以啊！明明大家都是大陆人！我也无所谓你怎么诅咒台湾，甚至我可以为了好玩和你一起骂台湾！但是让你正视国内党的独裁奴隶人民！你就说台湾！你不是脑子洗坏了就是欺软怕硬的垃圾人。你不觉得自己很可悲吗？

你个傻卵懂不懂对号入座 石砸狗叫啊？你个大陆吊毛代入台湾狗的视角说话？这台湾贱种是你射墙上出来的？你装你妈的白莲花呢？你妈的批也是粉的是吧？台湾脑残给人扣粉红帽子 你看不见，我支持把孽障屠杀殆尽你就出来装大陆仔了？你这算不算颠倒黑白？
跟个傻逼一样万物都扯到狗日的政治，怎么不见你爹把你射墙上也是政治逼的？

**Reply by frshgrass** [reply]
有毛病！啊q精神满分！都说了你怎么骂台湾我无所谓！劳资大陆人！你这个太搞笑了吧，自我催眠的垃圾人属于……

**Reply by frshgrass** [reply]
> > 你这个精神胜利法好可以啊！明明大家都是大陆人！我也无所谓你怎么诅咒台湾，甚至我可以为了好玩和你一起骂台湾！但是让你正视国内党的独裁奴隶人民！你就说台湾！你不是脑子洗坏了就是欺软怕硬的垃圾人。你不觉得自己很可悲吗？
> 
> 
> 
> 你个傻卵懂不懂对号入座 石砸狗叫啊？你个大陆吊毛代入台湾狗的视角说话？这台湾贱种是你射墙上出来的？你装你妈的白莲花呢？你妈的批也是粉的是吧？台湾脑残给人扣粉红帽子 你看不见，我支持把孽障屠杀殆尽你就出来装大陆仔了？你这算不算颠倒黑白？
> 
> 跟个傻逼一样万物都扯到狗日的政治，怎么不见你爹把你射墙上也是政治逼的？

有毛病！啊q精神满分！都说了你怎么骂台湾我无所谓！劳资大陆人！你这个太搞笑了吧，自我催眠的垃圾人属于……

**Reply by oceanzhang860411** [reply]
> > > 你这个精神胜利法好可以啊！明明大家都是大陆人！我也无所谓你怎么诅咒台湾，甚至我可以为了好玩和你一起骂台湾！但是让你正视国内党的独裁奴隶人民！你就说台湾！你不是脑子洗坏了就是欺软怕硬的垃圾人。你不觉得自己很可悲吗？
> > 
> > 
> > 你个傻卵懂不懂对号入座 石砸狗叫啊？你个大陆吊毛代入台湾狗的视角说话？这台湾贱种是你射墙上出来的？你装你妈的白莲花呢？你妈的批也是粉的是吧？台湾脑残给人扣粉红帽子 你看不见，我支持把孽障屠杀殆尽你就出来装大陆仔了？你这算不算颠倒黑白？
> > 跟个傻逼一样万物都扯到狗日的政治，怎么不见你爹把你射墙上也是政治逼的？
> 
> 有毛病！啊q精神满分！都说了你怎么骂台湾我无所谓！劳资大陆人！你这个太搞笑了吧，自我催眠的垃圾人属于……

支那豬把所有不同的聲音當成台灣人來意淫自己贏了，根本看不到共產黨拿著刀準備殺豬，我挺愛看的。建議韭菜多多贊鐮刀

**Reply by Cegazhos** [reply]
> > > 你这个精神胜利法好可以啊！明明大家都是大陆人！我也无所谓你怎么诅咒台湾，甚至我可以为了好玩和你一起骂台湾！但是让你正视国内党的独裁奴隶人民！你就说台湾！你不是脑子洗坏了就是欺软怕硬的垃圾人。你不觉得自己很可悲吗？
> > 
> > 
> > 你个傻卵懂不懂对号入座 石砸狗叫啊？你个大陆吊毛代入台湾狗的视角说话？这台湾贱种是你射墙上出来的？你装你妈的白莲花呢？你妈的批也是粉的是吧？台湾脑残给人扣粉红帽子 你看不见，我支持把孽障屠杀殆尽你就出来装大陆仔了？你这算不算颠倒黑白？
> > 跟个傻逼一样万物都扯到狗日的政治，怎么不见你爹把你射墙上也是政治逼的？
> 
> 有毛病！啊q精神满分！都说了你怎么骂台湾我无所谓！劳资大陆人！你这个太搞笑了吧，自我催眠的垃圾人属于……

那你反驳个毛啊？傻逼东西 知道自己讲话的逻辑么？没读几年书就多去读一读好不好？B作用没有在这叫

**Reply by frshgrass** [reply]
原来是脑子瓦特自贱的垃圾人呀。中国已经垃圾分类了，晚上回家要到正确的垃圾桶哦😘😘😘

**Reply by ghost** [reply]
> > > > 你这个精神胜利法好可以啊！明明大家都是大陆人！我也无所谓你怎么诅咒台湾，甚至我可以为了好玩和你一起骂台湾！但是让你正视国内党的独裁奴隶人民！你就说台湾！你不是脑子洗坏了就是欺软怕硬的垃圾人。你不觉得自己很可悲吗？
> 
> > > 
> 
> > > 
> 
> > > 你个傻卵懂不懂对号入座 石砸狗叫啊？你个大陆吊毛代入台湾狗的视角说话？这台湾贱种是你射墙上出来的？你装你妈的白莲花呢？你妈的批也是粉的是吧？台湾脑残给人扣粉红帽子 你看不见，我支持把孽障屠杀殆尽你就出来装大陆仔了？你这算不算颠倒黑白？
> 
> > > 跟个傻逼一样万物都扯到狗日的政治，怎么不见你爹把你射墙上也是政治逼的？
> 
> > 
> 
> > 有毛病！啊q精神满分！都说了你怎么骂台湾我无所谓！劳资大陆人！你这个太搞笑了吧，自我催眠的垃圾人属于……
> 
> 
> 
> 那你反驳个毛啊？傻逼东西 知道自己讲话的逻辑么？没读几年书就多去读一读好不好？B作用没有在这叫

真就操死你亲妈了，在这得逼得逼。你tm简直比共产党网评员还孝顺，是不是你全家都是共产党的狗奴才？村官天天拿着你妈当性奴玩？在你妈的阴道进进出出，最后把精液射在你妈嘴里？伴君如伴虎，要小心哦，当心你妈死无葬身之地，连骨灰都被扬了

**Reply by Cegazhos** [reply]
> > > > > 你这个精神胜利法好可以啊！明明大家都是大陆人！我也无所谓你怎么诅咒台湾，甚至我可以为了好玩和你一起骂台湾！但是让你正视国内党的独裁奴隶人民！你就说台湾！你不是脑子洗坏了就是欺软怕硬的垃圾人。你不觉得自己很可悲吗？
> > 
> > 
> > > >
> > 
> > 
> > > >
> > 
> > 
> > > > 你个傻卵懂不懂对号入座 石砸狗叫啊？你个大陆吊毛代入台湾狗的视角说话？这台湾贱种是你射墙上出来的？你装你妈的白莲花呢？你妈的批也是粉的是吧？台湾脑残给人扣粉红帽子 你看不见，我支持把孽障屠杀殆尽你就出来装大陆仔了？你这算不算颠倒黑白？
> > 
> > 
> > > > 跟个傻逼一样万物都扯到狗日的政治，怎么不见你爹把你射墙上也是政治逼的？
> > 
> > 
> > > 
> > 
> > 
> > > 有毛病！啊q精神满分！都说了你怎么骂台湾我无所谓！劳资大陆人！你这个太搞笑了吧，自我催眠的垃圾人属于……
> > 
> > 
> > 那你反驳个毛啊？傻逼东西 知道自己讲话的逻辑么？没读几年书就多去读一读好不好？B作用没有在这叫
> 
> 真就操死你亲妈了，在这得逼得逼。你tm简直比共产党网评员还孝顺，是不是你全家都是共产党的狗奴才？村官天天拿着你妈当性奴玩？在你妈的阴道进进出出，最后把精液射在你妈嘴里？伴君如伴虎，要小心哦，当心你妈死无葬身之地，连骨灰都被扬了

我支持民进党😘😘蔡英文是我妈也是你妈，你不该这么骂的

**Comment by hclgdb** [comment]
自己吃屎就说自己吃屎的问题，不要觉得别人吃了屎，自己吃也是对的。

**Reply by oceanzhang860411** [reply]
不不不，人家說的是，美國吃屎過，所以中國也要吃屎，而且還要吃更多，這樣才是中國夢

**Reply by hclgdb** [reply]
哈哈哈

**Comment by mxyulin** [comment]
油管去搜 翟山鹰-随心、随意、随笔
也就是地产党喊打喊杀的一个骗子老板（他确实承认骗了很多傻逼粉红的钱，利用傻逼粉红的一腔爱国心），翟山鹰父亲翟明战曾在空军军校最高任职副团、正团职参谋，空军大院红色系家庭。
他出国实质上是地产党要摘他公司多年来的财富和成果，结果和地产党没谈和，地产党想要杀人灭口，就听从挚友的劝诫跑到美国躲避政治迫害。
出于对中共地产党的恨意，他和王志安、郭文贵等反中共人士一样逐步披露中共地产党的种种奸邪淫恶。

**Reply by lance111l** [reply]
出国的不少是犯错的 怕被抓的

**Comment by lance111l** [comment]
停贷是经济问题 当然不是政治问题 开发商资金运作不行 倒闭了 美国倒闭的公司少吗

**Comment by Hcreak** [comment]
中国烂尾楼该美国屁事 你妈做饭难吃 你说隔壁小孩他妈做饭也难吃 都是隔壁小孩他妈做饭难吃害得你妈做饭也难吃？ 又赢麻了是吧
